packages/esm-patient-vitals-app/src/vitals/vitals.resource.tsx

```diff
@@ -11,11 +11,9 @@ export interface PatientVitals {
   systolic?: number;
   systolicRange?: any;
   diastolic?: number;
-  diastolicRange?: any;
   pulse?: number;
   temperature?: number;
   oxygenSaturation?: number;
-  oxygenSaturationRange?: any;
```

Comment:
Not used anywhere. Should have been removed in https://github.com/openmrs/openmrs-esm-patient-chart/pull/548/files.